### DISCUSSION TITLE
Does the official release binary package had tested on rhel8 or centos 8 ？

### DISCUSSION THREAD

**Discussion by vkingnew** [apache/doris#13423] [A - General / Q&A]
Does the official release binary package had tested on rhel8 or centos 8 ？
And current the official release binary package tested on which platforms ?

**Comment by dataalive** [comment]
mostly on centos 7.x,8.x, and ubuntu 18.04 , 20.04 on x86 machines and some arm platforms

**Comment by vkingnew** [comment]
the official docs show be modifed correctly.
``https://doris.apache.org/docs/install/install-deploy``
